salt/metalk8s/salt/minion/installed.sls

```diff
@@ -4,6 +4,15 @@ include :
   - metalk8s.repo
 
 Install salt-minion:
+{%- if grains['os'] == 'CentOS' %}
   {{ pkg_installed('salt-minion') }}
     - require:
       - test: Repositories configured
+
+{%- elif grains['os'] == 'Ubuntu' %}
+  pkg.installed:
+    - name: salt-minion
+    - require:
+      - test: Repositories configured
+
+{%- endif %}
```

Comment:
IMO you sould do the change in the macro `pkg_installed` directly as this change will be needed for all packages I think

Reply:
I change it in the macro but don't remove this part used in my first tests.
I will change it.